modules/github-oidc-role/additional-policy-map.tf

```diff
@@ -2,9 +2,10 @@ locals {
   # If you have custom policies, override this declaration by creating
   # a file called `additional-policy-map_override.tf`.
   # Then add the custom policies to the overridable_additional_custom_policy_map in that file.
-  # See the README in `aws-team-roles` for more details.
+  # The key should be the policy you want to override, the value is the json policy document.
+  # See the README in `github-oidc-role` for more details.
```

Comment:
OK, but now you need to explain this in the README. You can use https://github.com/cloudposse/terraform-aws-components/blob/main/modules/aws-team-roles/README.md#privileges-are-granted-to-users-via-iam-policies as a starting point, but don't just copy and paste because the details are different.